app/__init__.py

```diff
@@ -84,6 +84,13 @@ def load_member_locations():
         members = data['members']
     return members
 
+def load_member_hobby_data():
+    hobby_file_path = Path(__file__).resolve().parent / 'static' / 'data' / 'hobbies.json'
+    
+    with open(hobby_file_path, 'r') as f:
```

Comment:
Nit: It is always good to handle exceptions/errors like FileNotFoundError in this case.

Although, it should be fine for now, but it's a good thing to keep in mind